### ISSUE TITLE
cannot output pdf

### ISSUE THREAD

**Issue by Tariya321** [HFUTTUG/HFUT-Beamer#1]
Firstly, I cloned your project, and tried to run in my laptop but **failed to output pdf file, and termial printed words** as below:
`Package minted: You must invoke LaTeX with the -shell-escape flag`

So, I changed the latex based setting as 'pdf->bib->pdf->pdf'. And it really works though with a warning: 
`Package minted: You must have "pygmentize" installed to use this package.`

My Version is: TeX 3.141592653 (TeX Live 2022)

**Comment by HowardZorn** [comment]
> Firstly, I cloned your project, and tried to run in my laptop but **failed to output pdf file, and termial printed words** as below: `Package minted: You must invoke LaTeX with the -shell-escape flag`
> 
> So, I changed the latex based setting as 'pdf->bib->pdf->pdf'. 

You should not use the default settings from LaTeX Workshop to compile this project. We provide a customized configuration file for LaTeX Workshop which calls latexmk to make up the pdf file. Please read the README.

> And it really works though with a warning: `Package minted: You must have "pygmentize" installed to use this package.`

`minted` is use to print code snippets. If you don't need to show your code, you can remove it. Or you should install the pygments package: `pip3 install pygments`.

**Comment by Tariya321** [comment]
> You should not use the default settings from LaTeX Workshop to compile this project. We provide a customized configuration file for LaTeX Workshop which calls `latexmk` to make up the pdf file. Please read the README.
Yeah, my first attempt was based on your provided setting JSON file, which is placed at the next level of file **.vscode**.
Then some error (maybe……) happened and  pdf file cannot be outputed.

> `minted` is use to print code snippets. If you don't need to show your code, you can remove it. Or you should install the pygments package to have `pygmentize` in the `PATH`: `pip3 install pygments`.
I install this package, and it seems to be a doable solution for my situation.
The pdf. file has been outputed successfully!

I think that the output process was stopped by this error, and when I upgraded the python package, the process will continue, and then the pdf. file will be generated normally. By this case, it maybe better to add this situation into README.md because it has been ignored.

Whatever, thanks again! 👍
